packages-ts/starknet-gauntlet-starkgate/src/commands/bridge/setL2Token.ts

```diff
@@ -0,0 +1,65 @@
+import { BN } from '@chainlink/gauntlet-core/dist/utils'
+import {
+  AfterExecute,
+  BeforeExecute,
+  ExecuteCommandConfig,
+  ExecutionContext,
+  makeExecuteCommand,
+  Validation,
+} from '@chainlink/starknet-gauntlet'
+import { shortString } from 'starknet'
+import { isContext } from 'vm'
```

Comment:
```suggestion
```
These don't seem to be used. I also see them on `setL1Bridge.ts` and `deploy.ts`. 
`BN` and some others from `starknet-gauntlet` either.